packages/plugin-core/src/views/DendronTreeViewV2.ts

```diff
@@ -63,14 +63,12 @@ export class DendronTreeViewV2 implements vscode.WebviewViewProvider {
     const start = process.hrtime();
```

Comment:
Tangential question - do we still need `DendronTreeView.ts` for anything? (The V1)

Reply:
yeah, its actually the default tree view. the webview version of this is lacking accessibility features and the ability to persist toggled items